ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/hooks/useNavigateWithScanConfig.ts

```diff
@@ -0,0 +1,28 @@
+import { useContext } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { generatePathWithQuery } from 'utils/searchUtils';
+
+import { ScanConfigurationsContext } from '../ScanConfigurationsProvider';
+
+const useScanConfigRouter = () => {
```

Comment:
The hook name also differs from its file name in `import` statements.

Which name communicates the meaning more clearly?